src/analyze/D3/eventCount/js/chart_script.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Created by xianyuel on 7/24/17.
+ */
+
+var courseChart = dc.pieChart("#chart-course");
+var weekdayChart = dc.barChart("#chart-weekday");
+var hourChart = dc.barChart("#chart-hour");
+var dateChart = dc.lineChart("#chart-date");
+var volumeChart = dc.barChart("#chart-range-date");
+var charts = [courseChart, weekdayChart, hourChart, dateChart, volumeChart];
+
+d3.json("login_time.json").get(function(error, data){
+
+    if (error) throw error;
+
+    // format raw data
```

Comment:
Each object in the JSON has only a timestamp property and a `course` property.  There could be more than one object with the same timestamp and course, so shouldn't they each have an `id` property, too?